tracer/src/Datadog.Trace/Configuration/ConfigurationSources/Telemetry/ConfigurationBuilder.cs

```diff
@@ -115,6 +122,78 @@ public string AsString(string defaultValue, Func<string, bool>? validator)
             return defaultValue;
         }
 
+        [return: NotNullIfNotNull(nameof(getDefaultValue))]
+        private string? AsString(Func<string>? getDefaultValue, Func<string, bool>? validator, bool recordValue, string openTelemetryKey, Func<string, ParsingResult<string>>? openTelemetryConverter)
+        {
+            var result = Source.GetString(Key, Telemetry, validator, recordValue)
+                      ?? (FallbackKey1 is null ? null : Source.GetString(FallbackKey1, Telemetry, validator, recordValue))
```

Comment:
nitpicky (and not something that needs to be addressed in this PR), but all of these `FallbackKey..` lines seem (to me at least) to be a tad hard to read and are getting noisy
Maybe something that we can take a look at later 🤷

Reply:
It's easier to read at the _callsite_ at least :D